Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
+_kdeconnect-cli() {
+  _kdeconnect-cli-devices-id() {
+    compadd $(kdeconnect-cli --list-available --id-only)
+  }
+
+  _arguments \
+    '(-l --list-devices)'{-l,--list-devices}'[List all devices]' \
+    '(-a --list-available)'{-a,--list-available}'[List available (paired and reachable) devices]' \
+    '--id-only[Make --list-devices or --list-available print only the devices id, to ease scripting]' \
+    '--refresh[Search for devices in the network and re-establish connections]' \
+    '--pair[Request pairing to a said device]' \
+    '--ring[Find the said device by ringing it.]' \
+    '--unpair[Stop pairing to a said device]' \
+    '--ping[Sends a ping to said device]' \
+    '--ping-msg:Same as ping but you can set the message to display:' \
+    '--share:Share a file to a said device:_files' \
+    '--list-notifications[Display the notifications on a said device]' \
+    '--lock[Lock the specified device]' \
+    '--send-sms:Sends an SMS. Requires destination:' \
+    '--destination:Phone number to send the message:' \
+    '(-d --device)'{-d,--device}':Device ID:_kdeconnect-cli-devices-id' \
+    '(-n --name)'{-n,--name}':Device Name:' \
+    '--encryption-info[Get encryption info about said device]' \
+    '--list-commands[Lists remote commands and their ids]' \
+    '--execute-command:Executes a remote command by id:' \
+    '(-k --send-keys)'{-k,--send-keys}'[Sends keys to a said device]' \
+    '--my-id[Display this device''s id and exit]' \
+    '(-h --help)'{-h,--help}'[Displays help.]' \
```

Comment:
--help style options often exclude all other options from following. So this perhaps might be `'(-)'` instead of `'(-h --help)'`. Same goes for other options like --version, --author and --license. I suspect there are also further cases of options that are mutually exclusive. Can you specify both a device name and ID?